### PR TITLE
add gpuarray to python2 buildbot

### PR DESCRIPTION
**Pull request by slefrancois**
The FAST_COMPILE buildbot exposes a few ops that don't have perform methods:
http://darjeeling.iro.umontreal.ca:8080/job/Theano_buildbot_python2/168/testReport/

### CODE REVIEW

**Comment by slefrancois** [comment]
jenkins test this

**Comment by nouiz** [comment]
All tests passed. In the message, you talk as if you expected some tests failure. Is that the case? I'm ok to merge, but I'm not sure if that is what you wanted.

**Comment by slefrancois** [comment]
I think it's good to merge. But we need to look at the new errors it exposes. The old libgpuarray buildbot never tested fast_compile.

**Comment by slefrancois** [comment]
jenkins test this on earlgrey?

**Comment by slefrancois** [comment]
jenkins test this on earlgrey

**Comment by slefrancois** [comment]
jenkins test earlgrey

**Comment by slefrancois** [comment]
jenkins test earlgrey

**Comment by slefrancois** [comment]
jenkins test earlgrey

**Comment by nouiz** [comment]
I'm ok to merge if you think it is good. I think we can ignore the earlgrey failures, but just in case, I let you merge.

**Comment by slefrancois** [comment]
They are unrelated tests. I'll merge.

File: .jenkins/jenkins_buildbot_python2_debug.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ export LD_LIBRARY_PATH=/usr/local/cuda/lib64:$LD_LIBRARY_PATH
 export LIBRARY_PATH=/usr/local/cuda/lib64:$LIBRARY_PATH
 
 GPUARRAY_CONFIG="Release"
-DEVICE=cuda0
```
Comment (lamblin) [inline .jenkins/jenkins_buildbot_python2_debug.sh]: `DEVICE` is only read by libgpuarray / pygpu, which do not understand `"cuda"` anyway, they need the full name.
I think it was needed for running tests on the right device, but I do not think it is necessary for Theano's tests.
If using `cuda` worked, then I think we can delete the whole line.

Comment (slefrancois) [inline .jenkins/jenkins_buildbot_python2_debug.sh]: It was used by the libgpuarray buildbot to do the cmake libgpurray and pygpu tests, which we don't do here. We can remove the line in another PR.

**Pull request merged**